scripts/generate.js

```diff
@@ -44,6 +53,43 @@ for (let component of components) {
   fs.writeFileSync(`${__dirname}/../build/${component.name}Icon.js`, fileContent);
 
   indexContent += `export ${component.name} from './${component.name}Icon';\n`;
+  
+  /*
+    Pushing the different formats of typescript definitions to their respective arrays.
+  */
+  defFileFillIns.declareIconClasses.push(`class ${component.name}Icon extends GenericMdiIcon {}`);
+  defFileFillIns.declareConsts.push(`const ${component.name}Icon: new() => MdiReact.${component.name}Icon;`);
```

Comment:
Actually, this is good. Since this is a Typescript declaration file, I used the keyword "declare" instead of directly exporting and Typescript knows what to do.